javacord-api/src/main/java/org/javacord/api/entity/server/Server.java

```diff
@@ -2043,9 +2098,12 @@ default CompletableFuture<Void> banUser(long userId) {
      * @param userId            The id of the user to ban.
      * @param deleteMessageDays The number of days to delete messages for (0-7).
      * @return A future to check if the ban was successful.
+     * @deprecated Use {@link #banUser(String, int, TimeUnit)} instead as
+     *             deleteMessageDays is deprecated by discord.
```

Comment:
```suggestion
     * @deprecated Use {@link #banUser(String, int, TimeUnit)} instead.
```
The same for the other deprecations